cfgov/v1/util/categories.py

```diff
@@ -0,0 +1,25 @@
+from v1.util import ref 
+
+import logging
+logger = logging.getLogger(__name__)
+
+def clean_categories(selected_categories):
+    """ This is a (hopefully) temporary solution for dealing w/ the fact
+    that we show Blog and Reports as options for filtering, but
+    aren't categories themselves. Rather, they consist of sub-categories,
+    but since we aren't showing those sub-categories to the end user,
+    selecting the parent category is equivalent to all of them
+    getting checked. The mapping of Blog and Reports to their
+    respective categories exists in cfgov/v1/util/ref.py
+    """
+    if not selected_categories:
+        return None
+    subcategories_dict = dict(ref.categories)
+    for unicorn in ['blog', 'newsroom', 'research-reports']:
+        if unicorn in selected_categories:
+            if unicorn == 'research-reports':
+                unicorn = 'Research Report'
+            for category in subcategories_dict[unicorn.title()]:
+                selected_categories.append(category[0].lower())
+    logger.info('Filtering by categories {}'.format(selected_categories))
+    return selected_categories
```

Comment:
Yeah good catch. So right now I'm doing both.  Originally, it just modified the list and didn't return anything.  But now I need it to return the categories for the way I'm using it in this PR.  Would it be better to create a new list, return that, and then assign it in `forms.py` ? The only thing with that is I'd be copying over one list to another, which seems sort of unnecessary. But I don't know what the convention is for this sort of thing.